bin/nodeshift

```diff
@@ -57,8 +57,12 @@ yargs
     type: 'string'
   })
   .options('nodeVersion', {
-    describe: 'the version of Node.js to use for the deployed application.',
+    describe: 'DEPRECATED - use imageTag instead.  The version of Node.js to use for the deployed application.',
     alias: 'n',
+    type: 'string'
+  })
+  .options('imageTag', {
+    describe: 'The version of the docker image to use for the deployed application.',
```

Comment:
It really should say "tag" instead of "version" :-)

Reply:
dang it :)